packages/plugin-client-netlify/src/index.ts

```diff
@@ -0,0 +1,34 @@
+import type { OnPreBuild } from '@netlify/build';
+import { buildPreviewBranchName } from '@xata.io/client';
+
+export const onPreBuild: OnPreBuild = async function ({ netlifyConfig, utils }) {
+  const { CONTEXT: context, BRANCH: branch, XATA_PREVIEW: preview } = netlifyConfig.build.environment;
+
+  if (context !== 'deploy-preview') {
+    console.log('Not a deploy preview, skipping Xata plugin');
+    return;
+  }
+
+  if (preview !== 'netlify') {
+    console.log('XATA_PREVIEW is not set to netlify, skipping Xata plugin');
+    return;
+  }
+
+  console.log(JSON.stringify(netlifyConfig, null, 2));
+
+  if (!branch) {
+    console.log('No git HEAD found, skipping Xata plugin');
+    return;
+  }
+
+  // TODO: get org from netlify config
+  const org = 'SferaDev';
```

Comment:
We are going to ask Netlify for an environment variable for this. Hardcoded for now for the demo